README.md

```diff
@@ -127,127 +127,205 @@ future state of the libraries and to discover projects for contribution.
-jsii, and other related tooling. It also is a process for reviewing and
-discussing the proposal and tracking its implementation. "Request for Comments"
-means a request for discussion and oversight about the future of the CDK and
-jsii from contributors and users. It is an open forum for suggestions,
-questions, and feedback.
-
-The process is intended to be as lightweight and reasonable as possible for the
-present circumstances. As usual, we are trying to let the process be driven by
-consensus and community norms, not impose more structure than necessary.
-
-The RFC process itself is subject to changes as dictated by the core team and
-the community. Proposals can include proposed changes to the RFC process itself
-to better serve contributors.
-
-## When to submit an RFC?
-
-You should consider using this process if you intend to make "substantial"
-changes to [AWS CDK](https://github.com/aws/aws-cdk),
-[jsii](https://github.com/aws/jsii), or related tools. Some examples that would
-benefit from an RFC are:
-
-- Any change to existing APIs that could break existing code.
-- The removal of existing features or public APIs.
-- The introduction of new idiomatic usage or conventions, even if they do not
-  include code changes to CDK or jsii themselves.
-- Changes to the documented contribution workflow.
-- Features that cross multiple construct libraries.
-- Additions or changes to framework capabilities.
-- Additions or changes to formal specifications like cloud assembly, tree.json,
-  jsii, etc.
-
-The RFC process is a great opportunity to get more eyeballs on your proposal
-before it becomes a part of a released version of CDK/jsii. Quite often, even
-proposals that seem "obvious" can be significantly improved once a wider group
-of interested people have a chance to weigh in.
-
-The RFC process can also be helpful to encourage discussions about a proposed
-feature as it is being designed, and incorporate important constraints into the
-design while it's easier to change, before the design has been fully
-implemented.
-
-If you submit a pull request to implement a new major feature without going
-through the RFC process, it may be closed with a polite request to submit an RFC
-first.
-
-Some changes do not require an RFC:
-
-- Bugfixes for known issues.
-- Additions only likely to be _noticed by_ other developers of CDK/jsii, invisible
-  to users of CDK/jsii.
-- Additions of missing L1 or L2 constructs. Unless the service and/or constructs
-  are especially complex or intentionally diverge from existing api design best
-  practices.
-
-If you're not sure whether your change requires an RFC, feel free to create an
+An RFC is a document that proposes a change to one of the projects led by the
+CDK team at AWS. *Request for Comments* means a request for discussion and
+oversight about the future of the project from maintainers, contributors and
+users.
+
+**When to submit an RFC?** Anyone can submit an RFC for *any* change, big or
```

Comment:
add: The goal of the RFC is to reduce risk and ambiguity. One question that you should be able to answer after an RFC is ready is how long will it take to implement this in production quality. Do you both feel you flushed out enough of the design to be able to provide a confident answer?

Reply:
Added